util/mklink.pl

```diff
@@ -55,8 +55,10 @@
 foreach $file (@files) {
     my $err = "";
     if ($symlink_exists) {
-	unlink "$from/$file";
-	symlink("$to/$file", "$from/$file") or $err = " [$!]";
+        if (!-l "$from/$file") {
```

Comment:
I discovered that on linux if you relink a file, the link source has a newer timestamp, and apparently `make` considers the link source timestamp, and not the target. So we need to avoid recreation of symlinks when possible.